content_scripts/hints.js

```diff
@@ -10,6 +10,12 @@ var Hints = (function(mode) {
             holder.hide();
         } else if (event.keyCode === KeyboardUtils.keyCodes.shiftKey) {
             flip();
+        } else if (event.sk_keyName.toLowerCase() === runtime.conf.hintGroupCycleKey) {
```

Comment:
Need also check `runtime.conf.hintGroups` here, which means this line should be `runtime.conf.hintGroups && event.sk_keyName.toLowerCase() === runtime.conf.hintGroupCycleKey`, otherwise `ff` will stop working when `hintGroups` is disabled.